.github/workflows/run_rc_validation.yml

```diff
@@ -16,6 +16,7 @@
 # under the License.
 
 # To learn more about GitHub Actions in Apache Beam check the CI.m
+# DO NOT USE THIS WORKFLOW AND WILL BE REMOVED SOON
```

Comment:
Can we just remove it in this PR?

Reply:
let me keep this for now to track what we used to run. I will remove this later after I have good coverage.